tests/PHPUnit/Listener/ClearLogAndCacheTestListener.php

```diff
@@ -5,12 +5,15 @@
 use Nette\Utils\FileSystem;
 use Nette\Utils\Finder;
 use Nette\Utils\Strings;
-use PHPUnit\Framework\BaseTestListener;
+use PHPUnit\Framework\TestListener;
+use PHPUnit\Framework\TestListenerDefaultImplementation;
 use PHPUnit\Framework\TestSuite;
 use SplFileInfo;
 
-final class ClearLogAndCacheTestListener extends BaseTestListener
+final class ClearLogAndCacheTestListener implements TestListener
 {
+    use TestListenerDefaultImplementation;
```

Comment:
Is there a way to avoid traits? They increase legacy polution

Reply:
No, see the Announcement https://phpunit.de/announcements/phpunit-7.html :disappointed:

Reply:
I see